profiles/features/musl/package.mask

```diff
@@ -361,3 +361,7 @@ sys-process/systemd-cron
 dev-lang/rust-bin
 mail-client/thunderbird-bin
 www-client/firefox-bin
+
+# masking polkit-pkla-compat on musl
```

Comment:
Please put this at the top of the file (below the header) and also see https://devmanual.gentoo.org/profiles/package.mask/index.html.

Reply:
Done, thanks!